google/resource_dns_record_set.go

```diff
@@ -77,6 +77,25 @@ func resourceDnsRecordSetCreate(d *schema.ResourceData, meta interface{}) error
 		},
 	}
 
+	if d.Get("type").(string) == "NS" {
```

Comment:
Can you add a comment explaining why this section needs to happen? (even just a link to the docs is probably fine)